src/Symfony/Bundle/FrameworkBundle/Resources/config/services.php

```diff
@@ -200,6 +200,14 @@ class_exists(WorkflowEvents::class) ? WorkflowEvents::ALIASES : []
             ])
             ->tag('routing.expression_language_function', ['function' => 'env'])
 
+        ->set('container.get_routing_service', \Closure::class)
```

Comment:
confusing name again as it's not the routing service itself. maybe "container.get_routing_condition_service"